src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -67,17 +66,20 @@ class NotebookLauncherContent extends Component {
   }
 
   async getCluster() {
-    const { namespace } = this.props
+    const { clusters, refreshClusters } = this.props
+
+    await refreshClusters()
 
     return _.flow(
-      _.filter({ googleProject: namespace, creator: getBasicProfile().getEmail() }),
       _.remove({ status: 'Deleting' }),
       _.sortBy('createdDate'),
       _.last
-    )(await Jupyter.clustersList())
+    )(clusters)
```

Comment:
Careful with the timing here: this value was extracted from props before the refresh happened, so it could be stale by this point.